doc/source/examples/01_batch_inference/batch_inference.ipynb

```diff
@@ -0,0 +1,329 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6fbc3e3c",
+   "metadata": {},
+   "source": [
+    "# Batch Inference with Ray Data\n",
```

Comment:
You're missing the install deps steps for the many model and batch inference examples.

Reply:
Added it for the MMT one. Batch inference doesn't have any not covered by ray-ml